lib/readline.js

```diff
@@ -145,7 +145,7 @@ Interface.prototype.question = function question(query, options, cb) {
     };
     options.signal.addEventListener('abort', onAbort, { once: true });
     const cleanup = () => {
-      options.signal.removeEventListener(onAbort);
+      options.signal.removeEventListener('abort', onAbort);
```

Comment:
Could you send another PR for this fix so it can be backported more easily?

Reply:
Created another PR(#45676) to resolve this issue. PTAL.

Reply:
@aduh95 Can I create PR for backport of #45676? If yes, is `v18.x-staging` branch proper?

Reply:
Thank you for kind explanation. It's very helpful for me 👍